kahuna/public/js/services/image-logic.js

```diff
@@ -0,0 +1,51 @@
+import angular from 'angular';
+
+import '../services/image-accessor';
+
+const imageLogic = angular.module('kahuna.services.image-logic', [
+    'kahuna.services.image-accessor'
+]);
+
+/**
+ * Helpers to apply business logic on image resources.
+ */
+imageLogic.factory('imageLogic', ['imageAccessor', function(imageAccessor) {
+
+    function canBeDeleted(image) {
+        return image.getAction('delete').then(action => !! action);
+    }
+
+    function canBeArchived(image) {
```

Comment:
minor: is it worth calling this `canBePersisted` to have a common language? (ideally, would be done everywhere though :neutral_face: )